data/model/user_timeline_event.py

```diff
@@ -29,6 +29,7 @@ class UserTimelineEventType(Enum):
     RECORDING_RECOMMENDATION = 'recording_recommendation'
     FOLLOW = 'follow'
     LISTEN = 'listen'
+    NOTIFICATION = 'notification'
```

Comment:
I think that's because the `follow` event is a relationship and hence in the [user_relationship_enum](https://github.com/metabrainz/listenbrainz-server/blob/508d5a055b09fcc89aabcfbedf846f92d60c6524/admin/sql/create_types.sql#L5). The `listen` event is not stored separately in the database and the remaining two are in the `user_timeline_event_type_enum`.